### PR TITLE
#162416730 Users Should be able to view all Articles

### PR DESCRIPTION
**Pull request by dnuwa**
#### What does this PR do?
Users Should be able to view all Articles
#### Description of Task to be completed?
 - Any user is able to view all article
 - A user is able to view recent articles
#### How should this be manually tested?
Go to `/articles`, to view all articles
Go to `/` to view 4 of the most recent articles
#### Any background context you want to provide?
No need to be signed in to view all articles
#### What are the relevant pivotal tracker stories?
[#162416730](https://www.pivotaltracker.com/story/show/162416730)
#### Screenshots

All articles page `/articles`

image

Popular articles page/ home page `/`

image

### CODE REVIEW

File: src/actions/articleActions.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+});
+
+export const fetchArticlesFailure = errorMessage => ({
+  type: ACTION_TYPE.FETCH_ARTICLES_FAILURE,
+  errorMessage,
+});
+
+const fetchArticlesThunk = () => dispatch => axios.get('https://ah-backend-thanos-staging.herokuapp.com/api/articles')
+  .then((response) => {
+    //   console.log(response.data.results);
```
Comment (judeinno) [inline src/actions/articleActions.js]: Remove the commented out code

File: src/components/Articles/Articles.scss [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      text-align: center;
+      padding: 2px;
+    }
+    .body-section {
+    }
+    .date-section {
+    }
+    .read-more {
+      border-radius: 5px;
+    //   border: 1px solid red;
```
Comment (judeinno) [inline src/components/Articles/Articles.scss:52]: Again remove the commented out code

**Review by judeinno — DISMISSED** [review]
Thanks for the changes I requested for.

**Review by brucemakallan — DISMISSED** [review]
We'll need to create a chore for the sidebar on the homepage to show popular articles. Otherwise your work looks good to me.

File: src/components/Articles/Articles.scss (hunk 2 above)
Comment (dnuwa) [reply src/components/Articles/Articles.scss:52]: thanx @judeinno this has been worked on.

File: src/actions/articleActions.js (hunk 1 above)
Comment (dnuwa) [reply src/actions/articleActions.js]: Commented code was removed

**Review by Sulaiman-Mozes — DISMISSED** [review]

**Review by brucemakallan — APPROVED** [review]
We can do a little more CSS later. The PR looks great otherwise.

**Comment by Sulaiman-Mozes** [comment]
Work on that error in  your console, I am unable to run your deployed version
`app.bundle.js:27 TypeError: Cannot read property 'slice' of undefined`

**Pull request merged**